src/app/page/admin/decks/deck/manageCards/admin.manage.cards.component.ts

```diff
@@ -26,6 +26,7 @@ export class AdminManageCardsComponent implements OnInit {
   private url: string;
   private sub: Subscription;
   public selectedItem: number;
+  public listOfCards: string = 'Loading...'
```

Comment:
The variable is about message - I would rename to 'listOfCardsMessage' or something with word 'Message'.